autoload/db/adapter/bigquery.vim

```diff
@@ -0,0 +1,82 @@
+let g:bq_default_params = {
+  \ 'use_legacy_sql': 'false',
```

Comment:
A bit prescriptive, but this is a sane default

Reply:
[The docs](https://cloud.google.com/bigquery/docs/bq-command-line-tool#adding_flags_to_bigqueryrc) suggest you can also achieve this default with `.bigqueryrc`. This means:

1. Any user that wants this default can easily opt-in.
2. Any user that _doesn't_ want it will find it difficult to opt-out, because the hard-coded option will override `.bigqueryrc`.

So my instinct here is _not_ to provide a default. I say this with zero knowledge of the history, utility, or popularity of the option itself.

Reply:
Good point, there's plenty of other standard non-vim ways to configure this 👍

Reply:
Removed